packages/react-data-grid/src/Cell.tsx

```diff
@@ -23,31 +24,49 @@ function Cell<R>({
   onRowSelectionChange,
   rowData,
   rowIdx,
-  scrollLeft
+  scrollLeft,
+  eventBus,
+  onRowClick,
+  onRowDoubleClick,
+  onDeleteSubRow,
+  onCellExpand,
+  getCellActions,
+  enableCellRangeSelection
 }: CellProps<R>) {
+  const position: Position = { idx, rowIdx };
+
+  function selectCell(openEditor?: boolean) {
```

Comment:
I added ``eventBus` prop as it easier to pass it down instead of all the event handlers. It also works better with memo as we do not have to use `useCallback` everywhere